statline_bq/utils.py

```diff
@@ -687,21 +607,19 @@ def convert_table_to_parquet(
     pq_path = Path(f"{out_dir}/{file_name}.parquet")
 
     # Dump each bag partition to json file
-    bag.map(json.dumps).to_textfiles(temp_json_dir / "*.json")
+    print(f"Dumping bag to textfiles for {file_name}")
+    bag.map(ndjson.dumps).to_textfiles(temp_json_dir / "*.json")
+    print(f"Finished dumping bag to textfiles for {file_name}")
     # Get all json file names with path
     filenames = sorted(glob(str(temp_json_dir) + "/*.json"))
```

Comment:
I prefer to use `Path` objects throughout, which has [`Path.glob`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob)

Reply:
replaced with `filenames = [f for f in temp_json_dir.glob("*.json")]`